.babelrc

```diff
@@ -1,5 +1,8 @@
 {
-  "presets": [
-    "env"
+  "plugins": [
+    [ "@babel/plugin-transform-react-jsx", {
+      "importSource": "preact",

```

Comment:
I believe we should use the `@bpmn-io/properties-panel/preact` if we want this to be on par with `bpmn-js-properties-panel`. Cf.  https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/rollup.config.js#L42

Reply:
```suggestion
      "importSource": "@bpmn-io/properties-panel/preact",
```